webplugin/template/chat.html

```diff
@@ -13,15 +13,25 @@
     <script type="text/javascript">
         let params = new URLSearchParams(window.location.search);
         let appId = params.get('appId');
+        let botIds = params.get('botIds')
+        let assignee = params.get('assignee')
         let referer = window.document.referrer;
 
         if (!appId) {
             document.getElementById('error-msg').innerText = "no app id"
         } else {
             (function (d, m) {
+                if(botIds && assignee ){
+                var defaultSettings = {
+                "defaultBotIds": [botIds], // Optional. Replace <BOT_ID> with your bot ID which you can find in bot section of dashboard. Include this parameter if you want to assign the conversation to a bot.
+                "defaultAssignee": assignee, // Replace <ASSIGNEE_ID> with your bot/agent ID  whom you're assigning the conversation.
+                "skipRouting": true
```

Comment:
ideally this should be optional
default value being true

Reply:
My understanding was this if we make it true then only it will skip the Dashboard routing rules then and use the assignees defined here - 
if we make it false or eradicate - will the setting still apply?

Reply:
> if we make it false or eradicate - will the setting still apply?
Please check by keeping it false and update accordingly

Reply:
if it is false - it routes to default, we will need to keep it as True @prochnost04

Reply:
okay